<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetMainWid</class>
 <widget class="QWidget" name="SetMainWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>20</y>
     <width>831</width>
     <height>430</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>50</y>
     <width>152</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
border:5px solid rgb(15,78,225);
border-radius:15px
};</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="nameBtn">
      <property name="text">
       <string>名称设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="lineBtn">
      <property name="text">
       <string>主路设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="loopBtn">
      <property name="text">
       <string>支路设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="temBtn">
      <property name="text">
       <string>温度设置</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sysBtn">
      <property name="text">
       <string>系统设置</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
